quickstarts/gigamon-appinsights/config.yml

```
@@ -0,0 +1,49 @@
+
+
+slug: gigamon-appinsights
+
+title: Gigamon Application Insights
+
+description: |
+  ## Deep Application Visibility
+  Analytics tools are only as smart as the data they receive. Gigamon Application Metadata Intelligence empowers 
+  New Relic with critical metadata attributes across thousands of business, consumer and IT applications and services. 
+  Get deep application visibility to rapidly pinpoint performance bottlenecks, quality issues and potential network security risks.
+
+summary: |
+  Powered by Gigamon's Application Metadata Intelligence (AMI), part of Application Intelligence, to help you monitor and manage complex digital initiatives.
+
+level: Verified
+
+authors:
+  - Gigamon
+
+keywords:
+  - gigamon
+  - application
+  - network
+  - AMI
+  - intelligence
+  - metadata
+  - siem
+  - newrelic partner
+  - NR1_addData
+  - NR1_sys
+
+documentation:
+  - name: Application Metadata Intelligence
+    url: www.gigamon.com/products/optimize-traffic/application-intelligence/application-metadata
```

I realized it's looking for a full URL here, so adding the `https://` should make it happy
```suggestion
    url: https://www.gigamon.com/products/optimize-traffic/application-intelligence/application-metadata
```